mypyc/genops.py

```diff
@@ -306,14 +361,59 @@ def visit_import_all(self, node: ImportAll) -> Value:
 
         return INVALID_VALUE
 
-    def gen_func_def(self, fdef: FuncDef, class_name: Optional[str] = None) -> FuncIR:
+    def gen_glue_method(self, sig: FuncSignature, target: FuncIR,
+                        cls: ClassIR, base: ClassIR, line: int) -> FuncIR:
+        """Generate glue methods that mediate between different method types in subclasses.
+
+        For example, if we have:
+        class A:
```

Comment:
Style nits: Add empty lines between discussion and code example (both before and after code example) for better readability.